packages/react-components/review/beta/react-components.api.md

```diff
@@ -505,6 +505,7 @@ export const _DrawerMenu: (props: _DrawerMenuProps) => JSX.Element;
 export interface _DrawerMenuItemProps {
     disabled?: boolean;
     iconProps?: IIconProps;
+    id?: string;
```

Comment:
@JamesBurnside , I had to add an id prop for _DrawerMenuItemProps to pass down an id to the drawer menu items.

Reply:
Thanks for calling this out, looks perfect!